tests/e2e/tests/pilot/testdata/networking/v1alpha3/destination-rule-c-private.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: destination-rule-c
+spec:
+  host: c
+  {{if eq .globalMTlsEnable "true"}}
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+  {{end}}
+  subsets:
+    - name: v1
+      labels:
+        version: v1
+    - name: v2
+      labels:
+        version: v2
+  configScope: 1
```

Comment:
configScope: PRIVATE or PUBLIC. Enums in proto yamls can use the enum name

Reply:
ok, learned.